components/places/android/src/main/java/mozilla/appservices/places/Bookmarks.kt

```diff
@@ -4,7 +4,8 @@
 
 package mozilla.appservices.places
 
-import java.lang.RuntimeException
+import mozilla.appservices.places.uniffi.ReadonlyBookmarkItem
```

Comment:
Not a huge fan of `ReadonlyBookmarkItem` TBH - maybe just `BookmarkItem`? The fact we have that `Updatable` variant seems OK and is enough to differentiate IMO. This is hugely subjective though, so feel free to stick with what you prefer. :)

Reply:
I went back and forth myself